domain/cloudimagemetadata/state/types.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state
+
+import "time"
+
+// inputMetadata represents metadata information for a cloud image.
+// It is used to prepare and execute sqlair statement.
```

Comment:
Try to rephrase these comments without "It is used to prepare and execute sqlair statement."